storybook/main.js

```diff
@@ -17,4 +21,20 @@ module.exports = {
 		'@storybook/addon-viewport',
 		'@storybook/addon-a11y',
 	],
+	// Workaround:
+	// https://github.com/storybookjs/storybook/issues/12270
+	webpackFinal: async ( config ) => {
+		// Find the DefinePlugin
+		const plugin = config.plugins.find(
+			( p ) => p.definitions?.[ 'process.env' ]
```

Comment:
This is causing an error for me when I run `storybook:dev`:

```
[1] ERR!  /Users/sara/projects/gutenberg/storybook/main.js:29
[1] ERR!                        ( p ) => p.definitions?.[ 'process.env' ]
[1] ERR!                                               ^
[1] ERR! 
[1] ERR! SyntaxError: Unexpected token '.'
[1] ERR!     at wrapSafe (internal/modules/cjs/loader.js:915:16)
[1] ERR!     at Module._compile (internal/modules/cjs/loader.js:963:27)
[1] ERR!     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
[1] ERR!     at Module.load (internal/modules/cjs/loader.js:863:32)
[1] ERR!     at Function.Module._load (internal/modules/cjs/loader.js:708:14)
[1] ERR!     at Module.require (internal/modules/cjs/loader.js:887:19)
[1] ERR!     at require (internal/modules/cjs/helpers.js:74:18)
```

Reply:
Interesting! It was fine for me. Lemme fix :)